ui/src/terraui/build.sh

```diff
@@ -17,6 +17,9 @@ if [[ -e $REPODIR ]]; then
   if [ $? -eq 0 ]; then
     echo Repo exists and is current. Delete "$WD"/$REPODIR to rebuild.
```

Comment:
I always see "is current" regardless of whether that's true.  We should fix this, but we need to merge this PR ASAP so that can wait.